Advanced function usage/task3.js

```diff
@@ -0,0 +1,37 @@
+function divide (numerator, denominator){
+    if (denominator === 0){
+        throw new Error ("Cannot divide by zero");
+    } else if (isNaN(numerator) || isNaN(denominator)) {
+        throw new Error ("Use numbers only");
+    } else {
+        console.log(numerator / denominator);  
+    }
+}
+
+try {divide ( 1, 0);
```

Comment:
Дотримуйся пробілів і переносів стрічки, особливо якщо пристуні фігурні дужки. Бо читати неможливо.
```suggestion
try {
    divide (1, 0);
```